frontend/src/domain/entities/regulatory.js

```diff
@@ -177,6 +177,7 @@ export const FRANCE = 'Réglementation France'
 export const UE = 'Réglementation UE'
```

Comment:
Why not gather these labels into named objects like `REGULATION_COUNTRY_LABEL`, `REGULATION_ZONE_LABEL`, etc, since they are related? Single string global constants are usually quite seldom.

Reply:
This will need plenty of changes in backoffice and other part of the app, I prefer not to refactor that in this PR